app/views/idv/gpo_verify/index.html.erb

```diff
@@ -7,7 +7,11 @@
       ) %>
 <% end %>
 
+<% if @user_did_not_receive_letter %>
+<% title t('idv.gpo.did_not_receive_letter.title') %>
+<% else %>
 <% title t('idv.gpo.title') %>
```

Comment:
too late now but I would definitely indent this next time
```suggestion
  <% title t('idv.gpo.did_not_receive_letter.title') %>
<% else %>
  <% title t('idv.gpo.title') %>
```

Reply:
@jmdembe Merging this suggestion might cause merge conflicts for the next deploy, I'd suggest leaving it out (reverting 5c107a7)